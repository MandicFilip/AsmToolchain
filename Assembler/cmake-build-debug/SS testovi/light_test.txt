.data
a:
.word 4
b: .word 2

.text

fja:
    moveq psw , 0
    reteq

main:
    moveq r0, a
    moveq r1, $b
    muleq r1, r0
    diveq r1, r0

    jmpeq $fja
    moveq r7, -1
