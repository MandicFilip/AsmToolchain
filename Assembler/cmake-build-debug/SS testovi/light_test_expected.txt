
---Symbol Table---
Symbol   Id   Global   Defined    Value   Section    Offset
UND      0    0        1          0       UND        0
.data    1    0        1          a       data       0
.text    2    0        1          e       text       0
a        3    0        1          a       data       0
b        4    0        1          c       data       2
fja      5    0        1          e       text       0
main     6    0        1          14      text       6

#ret.data

#.ret.text
0008 apsolute 0001
000c relative 0001

#.data
04 00 02 00
#.text
e0 34 00 00 e0 29 10 35 00 00 30 35
00 00 28 09 28 0d f0 01 ea ff e0 35
ff ff
